### ISSUE TITLE
Property type 'JSQMessagesCollectionView *' is incompatible with type 'UICollectionView *' inherited from 'UICollectionViewLayout'

### ISSUE THREAD

**Issue by datomnurdin** [jessesquires/JSQMessagesViewController#960]
I got this warning message after updated latest pod.

```
/Users/MNurdin/Documents/iOS/xxxxx/Pods/JSQMessagesViewController/JSQMessagesViewController/Layout/JSQMessagesCollectionViewFlowLayout.h:59:60: Property type 'JSQMessagesCollectionView *' is incompatible with type 'UICollectionView *' inherited from 'UICollectionViewLayout'
```

Code

```
@property (readonly, nonatomic) JSQMessagesCollectionView *collectionView;
```

Please advice. Thank you.

**Comment by jessesquires** [comment]
fixed in 7.0.1. duplicate of #920

**Comment by jessesquires** [comment]
please don't open duplicate issues.
